index.html

```diff
@@ -27,12 +27,21 @@
     <header>
       <a href="#" class="title">Bilal Ahmed</a>
       <nav>
-        <img src="./img/threelines.png" class="menu" alt="menu bar" />
+        <ul class="Navigation">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><img src="./img/mail.png" alt="mail" /></li>
+        </ul>
       </nav>
+      <img src="./img/threelines.png" class="menu" alt="menu bar" />
     </header>
-    <section>
+    <section class="section-1">
```

Comment:
- [Opional] Though the previous reviewer also made the changes here optional, I also strongly advise that you fix them.  the horizontal scroll bar on the desktop version of your website content is overlapping with the website view width which is not a good user experience. You may want to check your wrapped `CSS` styling concerning this section. 

# View:
![image](https://user-images.githubusercontent.com/50990925/215013577-afa0e019-7cd9-43da-89a3-a0e7d4360d7f.png)

Reply:
Thanks for the review I tried but things messing up with mobile version code I have no idea how to solve the problem can you please check where I am making mistake ?

Reply:
hey I have fixed the issue